frontend-react/src/components/header/SupportDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import { Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import { useEffect, useState } from "react";
+import { NavLink } from "react-router-dom";
+
+const SupportDropdown = () => {
```

Comment:
Gonna reach out about not duplicating the dropdowns and instead using a new component from #5601 to create this.